crutest/src/main.rs

```diff
@@ -454,31 +439,23 @@ async fn get_region_info<T: BlockIO + Send + Sync + 'static>(
         max_block_io = total_blocks;
     }
 
-    println!(
-        "Region: es:{} ec:{} bs:{}  ts:{}  tb:{}  max_io:{} or {}",
-        extent_size.value,
-        total_blocks as u64 / extent_size.value,
-        block_size,
-        total_size,
-        total_blocks,
-        max_block_io,
-        (max_block_io as u64 * block_size),
-    );
-
     /*
      * Create the write log that tracks the number of writes to each block,
      * so we can know what to expect for reads.
      */
     let write_log = WriteLog::new(total_blocks);
 
-    Ok(RegionInfo {
+    let ri = RegionInfo {
         block_size,
-        extent_size,
+        sub_volume_info: extent_info_vec,
         total_size,
         total_blocks,
         write_log,
         max_block_io,
-    })
+    };
+    print_region_description(&ri, is_encrypted);
```

Comment:
It seems weird to print the region description as a side effect here (and this is the only part of the function which uses `is_encrypted`); could we make `print_region_description` the callers' responsibility?